src/tsconfig.json

```diff
@@ -1,49 +1,48 @@
 {
-    "version": "1.6.2",
-    "compilerOptions": {
-        "target": "es5",
-        "module": "commonjs",
-        "declaration": false,
-        "diagnostics": true,
-        "experimentalDecorators": true,
-        "emitDecoratorMetadata": true,
-        "isolatedModules": false,
-        "listFiles": true,
-        "noImplicitAny": false,
-        "noLib": false,
-        "noResolve": false,
-        "preserveConstEnums": true,
-        "removeComments": false,
-        "suppressImplicitAnyIndexErrors": true,
-        "sourceMap": true
-    },
-    "files": [
-        "BundleCompiler.ts",
-        "BundleParser.ts",
-        "CompileStream.ts",
-        "Compiler.ts",
-        "CompilerError.ts",
-        "CompilerHost.ts",
-        "CompilerReporter.ts",
-        "CompilerResult.ts",
-        "CompilerStatistics.ts",
-        "DependencyBuilder.ts",
-        "DiagnosticsReporter.ts",
-        "EventEmitter.ts",
-        "Glob.ts",
-        "Logger.ts",
-        "Project.ts",
-        "TsCore.ts",
-        "TsProject.ts",
-        "TsVinylFile.ts",
-        "Utilities.ts",
-        "references.d.ts"    ],
-    "bundles": {
-        "app": {
-            "files": [ "tsproject.ts", "references.d.ts" ],
-            "config": { 
-                "outDir": "./bundles"  
-            }
-        }
-    }
-}
+  "version": "1.6.2",
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "declaration": false,
+    "diagnostics": true,
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "isolatedModules": false,
+    "listFiles": true,
+    "noImplicitAny": false,
+    "noLib": false,
+    "noResolve": false,
+    "preserveConstEnums": true,
+    "removeComments": false,
+    "suppressImplicitAnyIndexErrors": true,
+    "sourceMap": true
+  },
+  "filesGlob": [
+    "./**/*{.ts, .d.ts}",
+    "../typings/tsd.d.ts",
+    "../node_modules/typescript/lib/typescript.d.ts"
+  ],
+  "files": [
+    "../typings/tsd.d.ts",
```

Comment:
Line 26: Why would tsd.d.ts be needed?
Line 27: Typescript 1.6.2 with module: commonjs should find typescript.d.ts